toolkit/README.md

```diff
@@ -1,8 +1,13 @@
-# Getting Started
 
-## [Quick Start](docs/building/building.md)
+## [Quick Start Guide](docs/quick_start/quickstart.md)
 
-- Install toolkit, build packages, build images
+- Quickly build CBL-Mariner, Install and Use
+
+## [Build Users Guide](docs/building/building.md) 
+
+- Build CBL-Mariner from End-to-End
```

Comment:
>from End-to-End [](start = 20, length = 15)

Nit: not an expert but I think it should be either "end-to-end" (no "from") or "from end to end" (no hyphens).

Reply:
Fixed